kuma/search/apps.py

```diff
@@ -12,9 +12,6 @@ class SearchConfig(AppConfig):
 
     def ready(self):
         """Configure kuma.search after models are loaded."""
-        # Register signal handlers
-        from . import signal_handlers  # noqa
-
         # Configure Elasticsearch connections for connection pooling.
         es_connections.configure(
```

Comment:
That's really interesting. I didn't think about that. In the olden Kuma, we used to create the connection to ES here. In my newen code, it's done in the view. Every time. Does that make a difference? I'll investigate.

Reply:
Actually, I've decided to keep that stuff. I just moved it. Now, instead of creating a new `Client` instance on every search, it uses `connections.configure()` and it seems to work. But it's breaking the mock testing :(
So I'll have to back to the drawing board there a little.